.devcontainer/devcontainer.json

```diff
@@ -24,7 +24,9 @@
       "installGradle": "false"
     },
     "ghcr.io/devcontainers/features/node:1": {},
-    "ghcr.io/devcontainers/features/python:1": {}
+    "ghcr.io/devcontainers/features/python:1": {},
+    "ghcr.io/devcontainers-contrib/features/pre-commit:2": {},
```

Comment:
**suggestion:** @avaleske just made some changes to remove contrib features - could we install these in the onCreateCommand from [this PR](https://github.com/civiform/civiform/pull/7415) instead?

Reply:
Thanks for the suggestion. Revised the approach and updated the PR description.